melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/joark/JournalpostMetadataService.java

```diff
@@ -0,0 +1,85 @@
+package no.nav.melosys.eessi.service.joark;
+
+import io.getunleash.Unleash;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.melosys.eessi.models.SedType;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+@Slf4j
+@Component
+public class JournalpostMetadataService {
+    private static final String UTLAND = "ab0313";
+    private static final String MEDLEMSKAP = "ab0269";
+    private final Unleash unleash;
+
+    public JournalpostMetadataService(Unleash unleash) {
+        this.unleash = unleash;
+    }
+
+    public JournalpostMetadata hentJournalpostMetadata(String sedType) {
+        if (!unleash.isEnabled("melosys.eessi.erstatte_dokkat") && Arrays.stream(SedType.values()).noneMatch(s -> Objects.equals(s.name(), sedType))) {
+            log.error("SedType {} er ikke støttet. Dersom vi skal støtte denne må den legges inn.", sedType);
+            return new JournalpostMetadata("", "");
+        }
+
+        return switch (SedType.valueOf(sedType)) {
+            case X001 -> new JournalpostMetadata("Anmodning om avslutning", UTLAND);
```

Comment:
Hvordan kan UTLAND eller MEDLEMSKAP være behandlingstema?
I S000001 er behandlingstema ab0269

Reply:
Tenkte det kanskje kunne skape forståelse for domenet, men kanskje ikke :P

Reply:
Det var veldig bra :rocket: 
Jeg ble forvirret fordi jeg ikke hadde sett UTLAND før